### PR TITLE
Creating a Form Using Plain Java

### PR DESCRIPTION
**Pull request by ruthstocks**
- Edited content (to conform to the book formatting).
- I have changed headers and text from "showing and hiding the FORM" to "showing and hiding the FORM DATA", which I think is more correct (but I could be wrong).

### CODE REVIEW

File: tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc [hunk 1]
```diff
@@ -40,36 +40,36 @@ private ComboBox<CustomerStatus> status = new ComboBox<>("Status");
...
 }
 ----
-<1> Adds all the enum values as options to the `ComboBox`.
+* `status.setItems` adds all the enum values as options to the `ComboBox`.
 
-<2> Makes the save button prominent by decorating it with a style name.
+* `addThemeVariants` makes the save button prominent by decorating it with a style name.
 
-== Showing and Hiding the Form
+== Showing and Hiding the Form Data
```
Comment (alejandro-du) [inline tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc]: @ruthstocks "Form Data" sounds wrong. What the tutorial is talking about is a "Form". Can you please fix before a merge this?

Comment (ruthstocks) [reply tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc]: Sorry about the delay. Fixing now.

File: tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc [hunk 2]
```diff
@@ -139,22 +140,22 @@ public CustomerForm(MainView mainView) {
...
-<2> Performs the save action in the backend.
+* `service.save(customer)` performs the save action in the backend.
 
-<3> Updates the list of customers in the main view.
+* `updateList` updates the list of customers in the main view.
 
-<4> Hides this form.
-
-Similarly, you can implement the delete action as follows:
+* `setCustomer(null)` hides the form data.
```
Comment (alejandro-du) [inline tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc]: @ruthstocks This and any other occurrence of "form data" should be reverted to "form" as well.

Comment (ruthstocks) [reply tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc]: Sorry, fixing now

Comment (alejandro-du) [reply tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc]: Thanks!

**Pull request merged**